docker-compose.yaml

```diff
@@ -41,6 +41,8 @@ services:
 
   db_psql:
     image: postgres
+    environment:
+      - POSTGRES_HOST_AUTH_METHOD=trust 
```

Comment:
what is this line doing

Reply:
Our previous setup allowed anyone to access our postgres db without a password. However a change in postgres has required passwords be provided. This env variable sets up the db with no password and trusts all access as we did before

I attempted to set the password to something that we could use but it appears I'd have to make a lot of modifications to default user setup code in order to get this to work. Since the `mysql` password is literally `test` I figured allowing this trust access is about equivalent security practices

Reply:
sad situation

Reply:
https://www.postgresql.org/docs/13/auth-trust.html

Looks like it opens up access to anyone that can connect to the port, right?

Is that sufficiently safe in this case?

Reply:
I believe that was the original behavior before postgres started requiring a password anyway? this only causes failures on the upgraded os alerting instances. not sure if I'm missing something

Reply:
Oh, my bad... I had the PR open but your comment hadn't appeared at the time.

Yeah, if this is just allowing the existing behavior, I suppose it's fine (though not ideal).